resources/views/frontend/pages/movie/home.blade.php

```diff
@@ -0,0 +1,227 @@
+                                                @foreach ($movie['types'] as $type)
+                                                        <a href="{{ route('frontend.catalog', ['type', $type['slug']]) }}">{{ trans($type['title']) }}</a>
+                                                @endforeach
+                                                </span>
+
+                                                <div class="card__wrap">
+                                                    <span class="card__rate"><i class="icon ion-ios-star"></i>{{ $movie['rate'] }}</span>
+                                                    <ul class="card__list">
+                                                        <li>{{ $movie['quality'] }}</li>
+                                                        <li>{{ $movie['age'] }}+</li>
+                                                    </ul>
+                                                </div>
+
+                                                <div class="card__description">
+                                                    <p>{{ $movie['description'] }}</p>
+                                                </div>
+                                            </div>
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+                        @endforeach
+                    <!-- end card -->
+                    </div>
+                </div>
+
+                <div class="tab-pane fade" id="tab-2" role="tabpanel" aria-labelledby="2-tab">
+                    <div class="row">
+                        @foreach ($tv_series as $tv_serie)
+                            <div class="col-6 col-sm-12 col-lg-6">
+                                <div class="card card--list">
+                                    <div class="row">
+                                        <div class="col-12 col-sm-4">
+                                            <div class="card__cover">
+                                                <img src="{{ asset($tv_serie['card_cover']) }}"
+                                                     alt="">
+                                                <a href="{{ route('frontend.movie',[$tv_serie['slug'], 1, 1]) }}" class="card__play">
+                                                    <i class="icon ion-ios-play"></i>
+                                                </a>
+                                            </div>
+                                        </div>
+
+                                        <div class="col-12 col-sm-8">
+                                            <div class="card__content">
+                                                <h3 class="card__title"><a
+                                                        href="{{ route('frontend.movie',[$tv_serie['slug'], 1, 1]) }}">{{ $tv_serie['name'] }}</a>
```

Comment:
```suggestion
                                                        href="{{ route('frontend.movie',[$tv_serie['slug'], 1, 1]) }}">{{ $tvSerie['name'] }}</a>
```
sửa tên biến từ controller